### PR TITLE
docs(agw): Updated AGW Install docs for the 1.8.0 Release

### PR DESCRIPTION
**Pull request by ShubhamTatvamasi**
Signed-off-by: Shubham Tatvamasi <shubhamtatvamasi@gmail.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by Neudrino — COMMENTED** [review]
- It would be really good if PRs would have a sensible description, including a test plan.
- To avoid that one has to constantly change this documentation, we might want to consider inserting a template string, e.g. `${VERSION}` or the like, with a (generic) comment, how that should be replaced.

**Comment by ShubhamTatvamasi** [comment]
> * It would be really good if PRs would have a sensible description, including a test plan.
> * To avoid that one has to constantly change this documentation, we might want to consider inserting a template string, e.g. `{VERSION}` or the like, with a (generic) comment, how that should be replaced.

We have not worked on any templates for the docs.

**Comment by Neudrino** [comment]
> > * It would be really good if PRs would have a sensible description, including a test plan.
> > * To avoid that one has to constantly change this documentation, we might want to consider inserting a template string, e.g. `${VERSION}` or the like, with a (generic) comment, how that should be replaced.
> 
> We have not worked on any templates for the docs.

I do not see how that would be in any relation or an answer to neither of the above points.

**Comment by ShubhamTatvamasi** [comment]
> > > * It would be really good if PRs would have a sensible description, including a test plan.
> > > * To avoid that one has to constantly change this documentation, we might want to consider inserting a template string, e.g. `${VERSION}` or the like, with a (generic) comment, how that should be replaced.
> > 
> > 
> > We have not worked on any templates for the docs.
> 
> I do not see how that would be in any relation or an answer to neither of the above points.

I guess we can do this once we have defined new structure.

**Comment by lucasgonze** [comment]
I have a request for the purpose of documenting secure deployment practices. Could you change this:

> - Download the Ubuntu Server 20.04 LTS `.iso` image from the Ubuntu website.

To this:

> - Download the Ubuntu Server 20.04 LTS `.iso` image from the Ubuntu website. Verify its integrity by checking [the hash](https://releases.ubuntu.com/20.04/SHA256SUMS).

**Comment by ShubhamTatvamasi** [comment]
> For [my own work item](https://github.com/magma/security/issues/104#issuecomment-1249944763) of documenting secure deployment practices, I'd like to submit additional edits whenever you're done with this.

I have included your changes @lucasgonze.

File: docs/readmes/lte/deploy_install.md [hunk 1]
```diff
@@ -94,7 +92,7 @@ journalctl -fu agw_installation
 When you see "AGW installation is done.", it means that your installation is complete. You can make sure magma is running by executing
 
 ```bash
-service magma@* status
+service 'magma@*' status
```
Comment (voisey) [inline docs/readmes/lte/deploy_install.md:95]: Why is this needed? It seems to be written without the quote marks throughout the repo.

File: docs/readmes/lte/deploy_install.md [hunk 2]
```diff
@@ -37,15 +35,15 @@ satisfies the following requirements:
...
+- Download the Ubuntu Server 20.04 LTS `.iso` image from the Ubuntu website. Verify its integrity by checking [the hash](https://releases.ubuntu.com/20.04/SHA256SUMS).
 - Create a bootable USB using this [Etcher tutorial](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos#0)
 - Boot your AGW host from USB
     - Press F11 to select boot sequence. WARNING: This might be different for your machine.
     - If you see two options to boot from USB, select the non-UEFI option.
 - Install and configure your Access Gateway according to your network defaults.
     - Make sure to enable ssh server and utilities (untick every other).
 - Connect your SGi interface to the internet and select this port during the
-installation process to get an IP using DHCP.
+installation process to get an IP using DHCP. (Consider enabling DHCP snooping to mitigate [Rogue DHCP](https://en.wikipedia.org/wiki/Rogue_DHCP))
```
Comment (voisey) [inline docs/readmes/lte/deploy_install.md:46]: How does one enable this in practice? It would be nice to give more specific instructions.

File: docs/readmes/lte/deploy_install.md (hunk 1 above)
Comment (lucasgonze) [reply docs/readmes/lte/deploy_install.md:95]: Because * will not be interpreted as a string literal. This will evaluate to different things depending on the directory where the command is being run. Adding single quotes ensures that the value is the same regardless of the context. It can't do damage because the shell will interpret them as metachars and remove them before passing in to the service params.

File: docs/readmes/lte/deploy_install.md (hunk 2 above)
Comment (lucasgonze) [reply docs/readmes/lte/deploy_install.md:46]: We could have the text "DHCP snooping" link to https://en.wikipedia.org/wiki/DHCP_snooping, like [DHCP snooping](https://en.wikipedia.org/wiki/DHCP_snooping).

File: docs/readmes/lte/deploy_install.md (hunk 1 above)
Comment (voisey) [reply docs/readmes/lte/deploy_install.md:95]: Okay, thanks for the clarification!

File: docs/readmes/lte/deploy_install.md (hunk 2 above)
Comment (voisey) [reply docs/readmes/lte/deploy_install.md:46]: That sounds good, thanks.

**Review by tmdzk — APPROVED** [review]

**Review by tmdzk — APPROVED** [review]

File: docs/readmes/lte/deploy_install.md (hunk 2 above)
Comment (voisey) [reply docs/readmes/lte/deploy_install.md:46]: This isn't a blocker so I will go ahead and approve this for now.

**Review by voisey — APPROVED** [review]
